### DISCUSSION TITLE
How to become admin for Mend Renovate Github integration?

### DISCUSSION THREAD

**Discussion by hmcguinn** [renovatebot/renovate#32409] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

Hi all! Please point me else-where if I'm in the wrong place. 

I recently setup [Renovate for a hosted Github app](https://docs.renovatebot.com/getting-started/installing-onboarding/#hosted-githubcom-app) and when going to the settings page for my organization `https://developer.mend.io/github/<ORG>/<REPO>/-/settings` see `Only users with admin permissions can change settings`. 

How are admin privileges defined? Is it linked to the Github repository permissions? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
It's linked to GitHub permissions

**Reply by hmcguinn** [reply]
Do you have any pointers to documentation on this?

**Reply by rarkins** [reply]
No, but there's not much details to share. If you are an admin for an org or repo on github, then the Mend app will set you control settings for the org/repo through the UI.
